frontend/dispatcher.lua

```diff
@@ -418,10 +420,11 @@ function Dispatcher:addItem(caller, menu, location, settings, section)
                             width = math.floor(Screen:getWidth() * 0.6),
                             value = location[settings] ~= nil and location[settings][k] or settingsList[k].default or 0,
                             value_min = settingsList[k].min,
-                            value_step = 1,
+                            value_step = settingsList[k].step or 1,
+                            precision = settingsList[k].precision,
```

Comment:
This probably `nil`s NumperPicker's default `precision` (`"%02d"`) if that is unset in `settingsList`, right?

Reply:
Yes I think so. But we could add an default precision "%02d" her eif you want.